src/components/common/Balance.js

```diff
@@ -0,0 +1,46 @@
+import React from 'react'
+
+// denomination of one near in minimal non divisible units (attoNears)
+// NEAR_NOMINATION is 10 ** 18 one unit
+const NOMINATION = 18
+const REG = /(?=(\B)(\d{3})+$)/g;
+
+const Balance = ({amount, milli}) => {
+    if (!amount) {
+        throw new Error("the account balance is wrong")
+    }
+    let index = amount.indexOf(".")
```

Comment:
in which case will amount string have dot?

Reply:
in the future, will user use near token to transfer or use string like "1111111111111111"? I just dont want to miss that situation

Reply:
not sure I understand.  Isn't this working on amount from backend (i.e. always integer)?